### PR TITLE
Update `Official beatmapping contest support` eligibility rules

### PR DESCRIPTION
**Pull request by MChecaH**
Changes discussed in [this internal thread](https://discord.com/channels/90072389919997952/1280086392931618857/1280086396840710165).

Tldr; Raw scoring has no benefit whatsoever over standard deviation for most cases. After reviewing multiple contests and seeing several hampered by using raw score, we're adding this change to avoid one single judge to have the power to harm one submission. This should better adhere to our current ruling to make all judges have the same amount of power.

## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] Waiting to announce things, prepare forum thread, osu! server announcement, and set a clear enforcement date. CC will then give their OK to merge this.

### CODE REVIEW

File: wiki/Contests/Official_support/en.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ tags:
 
 For other contests, email us at [tournaments@ppy.sh](mailto:tournaments@ppy.sh). We'll try to work something out.
 
-This article was last updated on July 10, 2024. For the full changelog, check out the [beatmapping contest support updates thread](https://osu.ppy.sh/community/forums/topics/1907886). For any queries or clarifications, please consult the `#tournaments` channel in the [osu! Discord server](https://discord.com/invite/ppy) or send an email to the [account support team](/wiki/People/Account_support_team) via [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
+This article was last updated on September 7, 2024. For the full changelog, check out the [beatmapping contest support updates thread](https://osu.ppy.sh/community/forums/topics/1907886). For any queries or clarifications, please consult the `#tournaments` channel in the [osu! Discord server](https://discord.com/invite/ppy) or send an email to the [account support team](/wiki/People/Account_support_team) via [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
```
Comment (cl8n) [inline wiki/Contests/Official_support/en.md:17]:
```suggestion
This article was last updated on 7 September 2024. For the full changelog, check out the [beatmapping contest support updates thread](https://osu.ppy.sh/community/forums/topics/1907886). For any queries or clarifications, please consult the `#tournaments` channel in the [osu! Discord server](https://discord.com/invite/ppy) or send an email to the [account support team](/wiki/People/Account_support_team) via [tournaments@ppy.sh](mailto:tournaments@ppy.sh).
```

File: wiki/Contests/Official_support/ko.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-outdated_since: 6d86d4792b0dc7ba98a33c8311254839229b3b10
+outdated_since: 11c4512d55442e331bd499af2d409c808ac482f1
```
Comment (cl8n) [inline wiki/Contests/Official_support/ko.md:2]:
```suggestion
outdated_since: 6d86d4792b0dc7ba98a33c8311254839229b3b10
```
dont change existing outdated_since

**Review by cl8n — CHANGES_REQUESTED** [review]

File: wiki/Contests/Official_support/en.md (hunk 1 above)
Comment (MChecaH) [reply wiki/Contests/Official_support/en.md:17]: Is there any specific, standardized reason to switch it around? That's how both the [Tournament support page](https://osu.ppy.sh/wiki/en/Tournaments/Official_support) and this one have been rolling by thus far.

File: wiki/Contests/Official_support/ko.md (hunk 2 above)
Comment (MChecaH) [reply wiki/Contests/Official_support/ko.md:2]: Deja vu

File: wiki/Contests/Official_support/en.md (hunk 1 above)
Comment (cl8n) [reply wiki/Contests/Official_support/en.md:17]: I guess how you have it is also fine (we use this for formatting <https://en.wikipedia.org/wiki/Wikipedia:Manual_of_Style/Dates_and_numbers#Formats>). but most newly-written articles prefer the british english version that I suggested, I think

**Review by cl8n — APPROVED** [review]
your choice to apply my other comment or not

File: wiki/Contests/Official_support/en.md (hunk 1 above)
Comment (MChecaH) [reply wiki/Contests/Official_support/en.md:17]: (In reference to https://github.com/ppy/osu-wiki/pull/12107#pullrequestreview-2287295494)

Letting it open so the other peeps in the committee check it out. If we want to follow the British version, we'll also change it in the other article on this same PR.

Comment (MChecaH) [reply wiki/Contests/Official_support/en.md:17]: Will leave as is at the moment. We can change it in a future update if the standard ever changes.

Comment (Hiviexd) [reply wiki/Contests/Official_support/en.md:17]: would stick w/ the same formatting the tournament support page currently uses for consistency. both can later be updated at the same time.

**Review by Hiviexd — APPROVED** [review]
